README.md

```diff
@@ -46,12 +46,15 @@ You need a POSIX-compatible environment to compile Lagrange.
 The required tools are a C11 compiler (e.g., Clang or GCC), CMake, `pkg-config`, and `zip`. Additional tools are required if HarfBuzz and GNU FriBidi are also compiled as part of the build (see next section for details).
 
 1. Download and extract a source tarball from [Releases][rel]. Please note that the GitHub/Gitea-generated tarballs do not contain HarfBuzz, GNU FriBidi, or [the_Foundation](https://git.skyjake.fi/skyjake/the_Foundation) submodules; check which tarball you are downloading. Alternatively, you may also clone the repository and its submodules: `git clone --recursive --branch release https://git.skyjake.fi/gemini/lagrange`
-2. Check that you have the recommended build tools and dependencies installed: SDL 2, OpenSSL 1.1.1, libpcre, libunistring, GNU FriBidi, and zlib. For example, on macOS this would do the trick (using Homebrew): ```brew install cmake sdl2 openssl@1.1 pcre libunistring fribidi``` Or on Ubuntu: ```sudo apt install cmake zip libsdl2-dev libssl-dev libpcre3-dev zlib1g-dev libunistring-dev libfribidi-dev```
-3. Optionally, install the mpg123 decoder library for MPEG audio support. For example, the macOS Homebrew package is `mpg123` and on Ubuntu it is `libmpg123-dev`.
+2. Check that you have the recommended build tools and dependencies installed: SDL 2, OpenSSL 1.1.1, libpcre, libunistring, GNU FriBidi, and zlib. For example,
+   * on macOS using Homebrew: `brew install cmake automake sdl2 openssl@1.1 pcre libunistring fribidi`
+   * on Ubuntu: `sudo apt install cmake zip libsdl2-dev libssl-dev libpcre3-dev zlib1g-dev libunistring-dev libfribidi-dev`
+   * on Fedora: `sudo dnf install cmake zip SDL2-devel openssl-devel pcre-devel zlib-devel libunistring-devel fribidi-devel`
+3. Optionally, install the mpg123 decoder library for MPEG audio support. For example, the macOS Homebrew package is `mpg123`, on Ubuntu it is `libmpg123-dev`, and on Fedora it is `mpg123-devel`.
 4. Create a build directory.
-5. In your empty build directory, run CMake: ```cmake {path_of_lagrange_sources} -DCMAKE_BUILD_TYPE=Release```
-6. Build it: ```cmake --build .```
-7. Now you can run *lagrange*, *lagrange.exe*, or *Lagrange.app*.
+5. In your empty build directory, run CMake to configure: `cmake .. -DCMAKE_BUILD_TYPE=Release`
+6. Build it: `cmake --build .`
+7. Now you can run _lagrange_, _lagrange.exe_, or _Lagrange.app_.
```

Comment:
FYI, the asterisk makes it more complicated to parse Markdown because of use for unordered lists and italics/bold at the same time